snippets/rmarkdown.json

```diff
@@ -71,7 +116,7 @@
 	"Insert code chunk": {
 		"prefix": "code chunk",
 		"body": [
-			"```{${1:language}}",
+			"```{${1|language,awk,bash,coffee,gawk,groovy,haskell,lein,mysql,node,octave,perl,psql,Rscript,ruby,sas,scala,sed,sh,stata,zsh,asis,asy,block,block2,bslib,c,cat,cc,comment,css,dot,embed,fortran,fortran95,go,highlight,js,julia,python,R,Rcpp,sass,scss,sql\\, connection=db,stan\\, output.var=\"stanmodel\",targets,tikz,verbatim|}}",
```

Comment:
I'm trying this, but I don't manage to see these languages anywhere in the completion list when I insert this snippet. Am I missing something?

Reply:
I'm thinking of adding this to the completion provider for R markdown.

Reply:
I revert this snippet for the moment for future revist.

Reply:
Looks like it is related to https://github.com/microsoft/vscode/issues/144362.

Reply:
Yes, it stopped working after the latest VSCode release. :(